packages/core/test/lib/transports/offline.test.ts

```diff
@@ -411,7 +416,7 @@ describe('makeOfflineTransport', () => {
     START_DELAY + 2_000,
   );
 
-  it.skip(
+  it(
```

Comment:
We previously skipped this test because it was flaky. I'm gonna un-skip it for now. Maybe using fake timers and or vitest helps. If we notice it being flaky, we should fix the test this time